boards/catie/zest_core_stm32h753zi/zest_core_stm32h753zi.dts

```diff
@@ -18,6 +18,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
+		zephyr,canbus = &fdcan1;
```

Comment:
Should the default fdcan interface be connected to 6TRON connector 1 instead of 6TRON conector 2, what do you think?

Reply:
it doesn't matter, logically it would be better. If we want to use it on connector 2, we will have to configure it in an app.overlay?